docs/tuf-spec.txt

```diff
@@ -530,6 +532,14 @@ Version 1.0 (Draft)
            , ... }
      }
 
+   COMPRESSION_ALGORITHM specifies one of the compression algorithms supported
```

Comment:
Like @JustinCappos, I'm also not sure that this change is straightforward.

This might have implications for snapshot metadata. There are use cases where, besides version numbers, the hashes of targets and delegated targets metadata files might be listed in the snapshot metadata. Do we include the hashes of both the compressed and uncompressed metadata files in there?